src/main/java/servletstask2/CalcServlet.java

```diff
@@ -0,0 +1,174 @@
+package servletstask2;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.util.function.ToIntBiFunction;
+import java.util.function.ToIntFunction;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+@WebServlet("/calc/*")
+public class CalcServlet extends HttpServlet {
+
+    @Override
+    protected void doPut(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        String var = req.getPathInfo().split("/")[1];
+
+        String val = req.getReader().readLine().replaceAll("\\s+", "");
+        if (var.equals("equation")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=5138103)